src/components/RegistrationPage.js

```diff
@@ -139,6 +147,7 @@ class RegistrationPage extends React.Component {
   }
 
   render() {
+    this.checkRegistration()
```

Comment:
Since this is being done in `render()`, it would probably be better to just return a `Redirect` component:

```jsx
import { Redirect } from 'react-router-dom'

// ...

if (!this.props.projectOpen) {
  return <Redirect to={`${process.env.PUBLIC_URL}/`} />
}
```